src/opts.rs

```diff
@@ -21,3 +21,4 @@ pub const PLUGIN_ALL_OPT: &str = "ALL";
 pub const PLUGIN_OVERRIDE_COMPATIBILITY_CHECK_FLAG: &str = "override-compatibility-check";
 pub const HELP_ARGS_ONLY_TRIGGER_TYPE: &str = "provide-help-args-no-app";
 pub const FROM_REGISTRY_OPT: &str = "REGISTRY_REFERENCE";
+pub const FERMYON_TOKEN: &str = "FERMYON_TOKEN";
```

Comment:
We should really rename _all_ the `HIPPO_*` auth vars rather than introducing a single different one. Maybe `SPIN_LOGIN_*`?

Reply:
renamed the FERMYON_TOKEN to token. about other renames, I think it is a bigger change as all the api/client at this moment refer to hippo. would it be ok if we defer that for a future PR?

Reply:
This name is used for the env var, so it should be given _some_ prefix.

Reply:
right 🤦, added `SPIN_LOGIN_` prefix for `TOKEN`. thank you Lann